testing/tests/DevExpress.ui.widgets/pager.tests.js

```diff
@@ -1632,14 +1632,12 @@ function() {
                 'dxPager-prevPage': 'test Previous Page',
                 'dxPager-nextPage': 'test Next Page',
             },
-            'ru': {
```

Comment:
Same here

Reply:
It is line before changes. It has been changed to 'fr'

Reply:
My bad :)